pkg/daemon/update.go

```diff
@@ -93,9 +93,56 @@ func getNodeRef(node *corev1.Node) *corev1.ObjectReference {
 	}
 }
 
-// finalizeAndReboot is the last step in an update(), and it can also
-// be called as a special case for the "bootstrap pivot".
-func (dn *Daemon) finalizeAndReboot(newConfig *mcfgv1.MachineConfig) (retErr error) {
+func reloadCrioConfig() error {
+	_, err := runGetOut("pkill", "-HUP", "crio")
```

Comment:
It would be nice if we could use `systemctl reload crio.service` instead. That way, crio can specify how exactly it needs to be reloaded (`ExecReload`) without the MCD needing to know the details. I don't think this will work today, since crio.service doesn't define an ExecReload (systemd will restart the service in this case).

Reply:
Agreed but I think for this release we are ok with this operation as it stands

Reply:
As Jerry said we can but we are trying to keep impact as minimal as possible. Since, crio supports [live configuration reload for container registry](https://github.com/cri-o/cri-o/blob/master/docs/crio.conf.5.md), we are making use of it in our usecase.